reference-manual/src/main/asciidoc/jet-concepts.adoc

```diff
@@ -576,7 +576,7 @@ maintain over a long term.
 
 ==== Caveats When Changing the Grouping Key
 
-You can change the key-extracting function for a `groupingKey`'s, but
+You can change the key-extracting function for a `groupingKey` 's, but
```

Comment:
I guess this doesn't look great rendered. Also, plural shouldn't use apostrophe. And plural shouldn't use "a". I think we can go with a singular and also add `()` to indicate it's a method:
```
for a `groupingKey()`.
```